src/Components/Blazor/Build/src/Core/RuntimeDependenciesResolver.cs

```diff
@@ -49,6 +51,11 @@ internal class RuntimeDependenciesResolver
             return paths.Concat(FindPdbs(paths));
         }
 
+        private static string GetAssemblyName(string entryPoint)
```

Comment:
Is `entryPoint` really the right name for this parameter? Should it be `filePath` or something like that?